config/application.yml.default

```diff
@@ -383,12 +383,12 @@ development:
   database_worker_jobs_host: ''
   database_worker_jobs_password: ''
   doc_auth_exit_question_section_enabled: true
-  doc_auth_selfie_capture: '{"enabled":false}'
   doc_auth_vendor: 'mock'
   doc_auth_vendor_randomize: false
   doc_auth_vendor_randomize_percent: 0
   doc_auth_vendor_randomize_alternate_vendor: ''
   doc_auth_not_ready_section_enabled: true
+  doc_auth_selfie_capture: '{"enabled": true}'
```

Comment:
I noticed you moved this down the list - if we're moving it, I wonder if we want to put these in alphabetical order? It seems to be the general pattern (although not 100% consistently done in this file).

Reply:
changed.